cmd/minishift/cmd/env.go

```diff
@@ -68,7 +68,7 @@ func getConfigSet(api libmachine.API, forceShell string, noProxy bool) (*DockerS
 		return nil, err
 	}
 
-	cmdLine := "minishift docker-env"
+	cmdLine := "minishift docker-env --no-proxy"
```

Comment:
This is wrong as you are string to add `--no-proxy` even if it is not selected by user. You should only append when `--no-proxy` when `--no-proxy` flag is passed.

Reply:
As commented: https://github.com/minishift/minishift/pull/1844#pullrequestreview-83717668